mta/mta_services.go

```diff
@@ -381,7 +380,7 @@ func RunAndWriteResultAndHash(info string, path string, action func() (interface
 	}
 	writeErr := WriteResult(result, hashcode, err)
 	if err != nil {
-		// The original error is more important
+		// The original error has greater importance and will be displayed first.
```

Comment:
Please use this message instead:

If there is an error in both the “ModifyMta” function and the  “WriteResult” function, only the “ModifyMta” function returns the error.